### PR TITLE
[doc] Add instruction for running cirque tests

### LINKED ISSUE #4084: [Doc] Add Cirque test document in chip master
We don't have cirque test document in chip so that we don't know how to locally reproduce it if it breaks.

### PR DESCRIPTION
**Pull request by erjiaqing**
#### Problem
We don't have cirque test document in chip so that we don't know how to locally reproduce it if it breaks.

 #### Summary of Changes
Add instruction for running cirque tests

 Fixes #4084

### CODE REVIEW

**Comment by erjiaqing** [comment]
Will merge restyled pr after this is approved.

**Review by rwalker-apple — APPROVED** [review]

**Review by yunhanw-google — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Pull request merged**
